test/jdk/java/lang/Class/getEnclosingClass/EnclosingClassTest.java

```diff
@@ -76,23 +154,24 @@ static void info(Class<?> c, Class<?> encClass, String desc) {
                            c.getCanonicalName() + "'");
     }
 
-    static void match(String actual, String expected) {
-        assert((actual == null && expected == null) || actual.equals(expected));
+    private void match(String actual, String expected) {
+        System.out.println("actual:" + actual + "expected:" + expected);
+        assert((actual == null && expected == null) || actual.equals(expected.trim()));
```

Comment:
I don't think this check is done, because assertions aren't enabled.  This should be changed to explicitly throw an exception.